frontend/packages/kubevirt-plugin/integration-tests/tests/models/kubevirtDetailView.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-await-in-loop */
+import { selectDropdownOption, click } from '../../../../console-shared/src/test-utils/utils';
+import { isLoaded, resourceRows } from '../../../../../integration-tests/views/crud.view';
+import { TABS, diskTabCol, networkTabCol } from '../utils/consts';
+import { storageResource, networkResource } from '../utils/types';
+import { fillInput } from '../utils/utils';
+import * as kubevirtDetailView from '../../views/kubevirtDetailView.view';
+import { confirmAction } from '../../views/vm.actions.view';
+import { DetailView } from './detailView';
+
+export class KubevirtDetailView extends DetailView {
+  async getAttachedDisks(): Promise<storageResource[]> {
+    await this.navigateToTab(TABS.DISKS);
+    const resources = [];
+    for (const row of await resourceRows) {
+      const cells = row.$$('div');
+      resources.push({
+        name: await cells.get(diskTabCol.name).getText(),
+        size: (await cells.get(diskTabCol.size).getText()).match(/^\d*/)[0],
+        storageClass: await cells.get(diskTabCol.storageClass).getText(),
+      });
+    }
+    return resources;
+  }
+
+  async getAttachedNics(): Promise<networkResource[]> {
```

Comment:
Just a reminder that Console acronym naming convention is to use uppercase, e.g. `getAttachedNICs`.

In context of KubeVirt plugin package, we should pick one convention and stick with it consistently.

Reply:
Fixed.